universal-application-tool-0.0.1/conf/evolutions/default/15.sql

```diff
@@ -0,0 +1,11 @@
+# --- Add localized name and description to the Programs table.
+
+# --- !Ups
+
+alter table programs add localized_name jsonb;
+alter table programs add localized_description jsonb;
+
```

Comment:
Thought (blocking): is it okay for jsonb to be null? If not please provide default value for backward compatibility.

```sql
update programs set localized_name = '{"en_US" : "' || name || '"}' where localized_name is null;
```

Reply:
Good catch, thank you!  Added defaults for localized_name and localized_description